app/src/main/res/values/styles.xml

```diff
@@ -45,4 +57,29 @@
         <item name="android:background">@color/white</item>
     </style>
 
+    <!-- Register Screen -->
```

Comment:
These should be reused in the rest of the app where applicable, not just in this screen, so I don't think there's a need for this comment. Also, it might be more efficient to inherit more props from the existing text styles.